docker-compose.yml

```diff
@@ -1,10 +1,9 @@
-# docker-compose.yml
 version: '3'
 networks:
   todoapp-net:
 services:
   pythonapp:
-    image: todoapp:2.0.0
+    image: todoapp:${VERSION:-2.0.0}
```

Comment:
The image name `todoapp:${VERSION:-2.0.0}` assumes that the `VERSION` environment variable is set. If not, it defaults to `2.0.0`. Ensure that this image tag exists in your Docker registry.